spec/helpers/identity_account_helper_spec.rb

```diff
@@ -32,4 +32,23 @@
       expect(link).to match("sign_out_uri=#{url_encoded}")
     end
   end
+
+  describe "#identity_link_uri" do
+    let(:fake_acocunt_link) do
```

Comment:
typo

Reply:
And a rude one at that 🫣

Will fix right away!